cdap-master/pom.xml

```diff
@@ -149,6 +149,13 @@
       <version>${project.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
needed for the DatasetUpgraderTest. It needs a DatasetFrameworkTestUtil, which is from data fabric test jar